docker/renew_ldap.sh

```diff
@@ -0,0 +1 @@
+docker-compose rm -fs openldap && docker-compose up -d openldap
```

Comment:
Please add a newline at the bottom, POSIX standard mandates newline at the end of (any) text files.